core/src/test/java/org/testcontainers/junit/CopyFileToContainerTest.java

```diff
@@ -0,0 +1,39 @@
+package org.testcontainers.junit;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.startupcheck.MinimumDurationRunningStartupCheckStrategy;
+import org.testcontainers.utility.MountableFile;
+import java.io.File;
+import java.time.Duration;
+
+public class CopyFileToContainerTest {
+    private static String folderPath = new File("./src/test/resources/mappable-resource/").getAbsolutePath();
+    private static String containerPath = "/opt";
+    private static String fileName = "test-resource.txt";
+    public static GenericContainer container;
+
+    @Before
+    public void before() {
+        container = new GenericContainer("couchbase:latest")
+            .withStartupCheckStrategy(new MinimumDurationRunningStartupCheckStrategy(Duration.ofSeconds(5)))
+            .withCopyFileToContainer(MountableFile.forHostPath(folderPath), containerPath );
+    }
+
+
+    @Test
+    public void checkFileCopied() {
+        container.start();
+        try {
+            String filesList = container.execInContainer("ls","/opt/mappable-resource").getStdout();
+
+            Assert.assertTrue(filesList.contains(fileName));
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Please let the exception be thrown from the test method, as it could be important.